### PR TITLE
 Added a pronunciation guide to the word Xarray in the README.MD fil…

### PR DESCRIPTION
**Pull request by Chinemere**
Added a pronunciation guide to the word Xarray in the README.MD file so users can easily pronounce the word correctly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
...
 [![Twitter](https://img.shields.io/twitter/follow/xarray_dev?style=social)](https://twitter.com/xarray_dev)
 
-**xarray** (formerly **xray**) is an open source project and Python
+**xarray** (pronounced as ex-array, formerly **xray**) is an open source project and Python
```
Comment (TomNicholas) [inline README.md:14]:
```suggestion
**xarray** (pronounced "ex-array", formerly known as **xray**) is an open source project and Python
```

**Review by TomNicholas — COMMENTED** [review]
Thanks for this @Chinemere! It never occurred to me that this was ambiguous but it definitely is, thank you.

I've suggested a small change to the grammar - if you hit "commit" next to my change it will merge the change into your pull request.

**Review by Chinemere — COMMENTED** [review]
Thank you @TomNicholas, I appreciate your feedback, I am pretty new to contributing to open source, but I assure you that I am a fast learner.

In your comment, you said "If you hit commit next to my change, it will merge the change into your pull request "  kindly bear with me, I can't seem to find any button that says commit next to your change. 

I could only see a button that said review changes, then I made this comment, When I tried to click on the approve option, it said Pull request authors are not allowed to make approvals. so I just went ahead and clicked on review changes. 
I wouldn't know if this is enough to commit your changes to my pull request. Kindly let me know what you think. 

Moreover please is there a way I can reach out to you? I tried reaching out to you on Twitter and the provided email, but I have not been successful with that. The mail seems to be invalid.

I am trying to record my contributions to this project and I would greatly need your help to guide me on the timelines for this project and know the next action to be taken.

Thanks once again for all you do for the Open Source community, I am eagerly waiting for your response.

**Comment by TomNicholas** [comment]
> In your comment, you said "If you hit commit next to my change, it will merge the change into your pull request " kindly bear with me, I can't seem to find any button that says commit next to your change.

You can ignore this comment - I suggested a change, asked you to commit it, then realised it would be simpler for you if I just used my admin rights to commit it myself, and edited my comment to remove the recommendation. Sorry for the confusion.

> Moreover please is there a way I can reach out to you? I tried reaching out to you on Twitter and the provided email, but I have not been successful with that. The mail seems to be invalid.

Sorry! I've been getting a lot of messages from Outreachy applicants and have not managed to reply to them all :cry: The email should not be invalid though - where did you send it?

If you have questions about xarray itself they should be raised on the repository though - you don't need to contact me directly for that. Also if you raise them publicly on the repository it gives other people a chance to answer if I don't see it.

If you have questions about Outreachy specifically then ask them either on the discord channel or by emailing me (thomas dot nicholas at columbia dot edu).

**Comment by dcherian** [comment]
Thanks for your contribution @Chinemere !

> Moreover please is there a way I can reach out to you? I tried reaching out to you on Twitter and the provided email, but I have not been successful with that.

Note that open-source projects run on volunteer time (mostly) and it can be hard to respond very quickly.  It may even take a few days, so it's usually best to be patient and ping in a week or so if you haven't received a response.

**Pull request merged**